ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -40,54 +55,125 @@
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
   private final KsqlEngine engine = EasyMock.niceMock(KsqlEngine.class);
   private final UdfLoader udfLoader = EasyMock.niceMock(UdfLoader.class);
-  private final String query = "select * from bar;";
-  private StandaloneExecutor executor;
+  private String queriesFile;
 
   @Before
   public void before() throws IOException {
-    final String queriesFile = TestUtils.tempFile().getPath();
-    executor = new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader);
-    try(final FileOutputStream out = new FileOutputStream(queriesFile)) {
-      out.write(query.getBytes(StandardCharsets.UTF_8));
-    }
+    queriesFile = TestUtils.tempFile().getPath();
   }
 
   @Test
-  public void shouldCreateQueries() {
-    EasyMock.expect(engine.createQueries(query + "\n", ksqlConfig))
+  public void shouldRunStatements() throws IOException {
```

Comment:
Testing way too much. 1 test per statement. Then i can just read the test names to know which statements it can execute

Reply:
Broke down the test into multiple tests.